notebooks/filechecker/argo-file-checker.ipynb

```
@@ -0,0 +1,408 @@
+    "\n",
+    "# Local Docker instance\n",
+    "API_BASE_URL=\"http://localhost:8000\"\n",
+    "\n",
+    "# Kubernetes test instance\n",
+    "# API_BASE_URL= \"https://livkrakentst.clusters.bodc.me/ewetchy/amrit/argo-toolbox/api/file-checker\"\n",
+    "\n",
+    "# Default DAC for validation\n",
+    "DEFAULT_DAC = \"bodc\"\n",
+    "\n",
+    "# Mount location of all the deployment files\n",
+    "# if you are running the API locally via Docker, ensure this path matches the volume mount in your Docker setup\n",
+    "# for example docker  run --rm --name argo-file-checker2 -p 8000:8000 argo-file-checker\n",
+    "\n",
+    "# Result file for full deployment checks\n",
+    "deployments_files_check_result_file=\"results/deployment_files_check_result.csv\"\n",
+    "\n",
+    "# Endpoints\n",
+    "CHECK_FILE_ENDPOINT = \"/check-files\"\n",
+    "FULL_DEPLOYMENT_CHECK_ENDPOINT = \"/check-deployment\"\n",
+    "\n",
+    "# Request settings\n",
+    "TIMEOUT = 30  # API request timeout in seconds\n",
+    "HEADERS = {\n",
+    "    \"accept\": \"application/json\",\n",
+    "}\n",
+    "\n",
+    "# Full URL for file check endpoint\n",
+    "FILE_CHECK_URL = f\"{API_BASE_URL}/{CHECK_FILE_ENDPOINT}\"\n",
+    "\n",
+    "print(f\"🔗 API Base URL: {API_BASE_URL}\")\n",
+    "print(f\"🏛️ DAC: {DEFAULT_DAC}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da2e8bb9",
+   "metadata": {},
+   "source": [
+    "## Check the API connections."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e49a257d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ===============================\n",
+    "#  3️⃣ Check the API connection\n",
+    "# ===============================\n",
+    "try:\n",
+    "    response = requests.get(f\"{API_BASE_URL}/\", timeout=5)\n",
+    "    response.raise_for_status()\n",
+    "    print(\"✅ API is reachable.\")\n",
+    "except:\n",
+    "    print(\"❌ API is not reachable.\")\n"
```

I think we can skip this bit – all it really does is hide the error.